packages/react-core/src/components/Button/Button.tsx

```diff
@@ -70,10 +76,8 @@ export interface ButtonProps extends Omit<React.HTMLProps<HTMLButtonElement>, 'r
   icon?: React.ReactNode | null;
   /** Sets the button tabindex. */
   tabIndex?: number;
-  /** Adds small styling to the button */
-  isSmall?: boolean;
-  /** Adds large styling to the button */
-  isLarge?: boolean;
+  /** Adds small or large styling to the button */
+  customSize?: 'default' | 'small' | 'large';
```

Comment:
maybe we should just call this `size`.  @mcoker any thoughts on this?

Reply:
jenny also mentioned that, i just checked in the changes